securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -179,27 +179,14 @@ def _check_login_with_otp(self, otp):
         # Test that the new user was logged in successfully
         self.assertIn('Sources', self.driver.page_source)
 
-    def _check_login_with_skewed_otp(self):
-        interval = 30
-
-        # Client is behind server
-        otp = self.new_user['orm_obj'].totp.at(
-            datetime.datetime.now() - datetime.timedelta(seconds=interval))
-        self._check_login_with_otp(otp)
-
-        # Client is ahead of server
-        otp = self.new_user['orm_obj'].totp.at(
-            datetime.datetime.now() + datetime.timedelta(seconds=interval))
-        self._check_login_with_otp(otp)
-
```

Comment:
This test would always pass because it's mocked, and it looks like the underlying functionality - the `valid_window` option of `verify` which we now use for this check - is tested in `pytop`, so good to remove :+1: